src/components/Button/Button.module.scss

```
@@ -0,0 +1,36 @@
+.button {
+  display: block;
+  width: 100%;
+  color: var(--theme-color-white);
+  background: var(--theme-color-accent);
```

❗️
Оно не сочетается. Если где-то нейминг семантический, то он везде должен быть таким. Потом захочется изменить css-вары (меняем тему приложения через css-вары, мы к этому, вероятно, и придём чуть позже) и `--theme-color-white` надо будет менять, в итоге в переменной, где по смыслу должен находиться белый цвет, окажется другой. Семантику рулит)
```suggestion
  color: var(--theme-color-text);
  background: var(--theme-color-accent);
```

---

цвет текста же везде разный, где то --theme-color-white гдето --theme-color-basic, тогда придется наделать кучу переменных --theme-color-text --theme-color-buttoText --theme-color-buttonWarningtext на каждый модификатор и компонент
тема дает нам свои переменные а за то что при их изменении ничего не сломалось отвечает ее макет

---

темную тему нарисуем просто путем изменения переменных в макете и в верстке

---

Если где-то нужен просто белый цвет, который не меняется, то это не тема, это просто `--color-white`. В теме всё должно быть семантично